services/crawlingService.js

```diff
@@ -1,54 +1,361 @@
+  await driver.get(firstPageUrl);
+
+  const orderButtonList = await driver
+    .findElement(By.id('sort_btn'))
+    .findElement(By.tagName('ul'))
+    .findElements(By.tagName('li'));
+  const dateOrderButton = await orderButtonList[4].findElements(
+    By.tagName('a'),
+  );
+
+  await dateOrderButton[1].click();
 
-  const loginPageButton = await driver.findElement(By.className('logon')).findElement(By.tagName('a'));
+  let newUrl = firstPageUrl;
+  newUrl = newUrl.replace('page=1', 'page=0');
+
+  for (let i = 1; i < 31; i++) {
+    newUrl = newUrl.replace(`page=${i}`, `page=${i + 1}`);
+    await driver.get(newUrl);
+    await connectToAuctionDetailPage(driver);
+  }
+}
+
+async function connectToAuctionDetailPage(driver) {
+  const auctionListTable = await driver.findElement(
+    By.className('tbl_auction_list'),
+  );
+  const auctionList = await auctionListTable.findElements(By.tagName('tr'));
+  const auctionBuildingLink = [];
+
+  for (let i = 1; i < auctionList.length; i++) {
+    await driver.wait(until.elementLocated(By.tagName('td')), 10000);
+
+    const auctionListDataSells = await auctionList[i].findElements(
+      By.tagName('td'),
+    );
+    auctionBuildingLink.push(
+      await auctionListDataSells[1]
+        .findElement(By.tagName('a'))
+        .getAttribute('href'),
+    );
+  }
+
+  for (let i = 1; i < auctionList.length; i++) {
+    await driver.get(auctionBuildingLink[i - 1]);
+    await crawlingAuctionDetail(driver);
+    await driver.navigate().back();
+  }
+}
+
+async function loginAuctionPage(driver) {
+  const loginPageButton = await driver
+    .findElement(By.className('logon'))
+    .findElement(By.tagName('a'));
   await loginPageButton.click();
 
   const loginInput = await driver.findElement(By.className('mem_id'));
-  await loginInput.sendKeys("vanilla")
+  await loginInput.sendKeys('vanilla');
   const passwordInput = await driver.findElement(By.className('mem_pw'));
-  await passwordInput.sendKeys("asdasd");
+  await passwordInput.sendKeys('asdasd');
   const loginButton = await driver.findElement(By.id('btn_login'));
   await loginButton.click();
+}
 
-  const orderButtonList = await driver.findElement(By.id('sort_btn')).findElement(By.tagName('ul'))
-    .findElements(By.tagName('li'));
-  const dateOrderButton = await orderButtonList[4].findElements(By.tagName('a'));
-  await dateOrderButton[1].click();
+async function connectSetAuctionSearchOptions(driver) {
+  await driver.get('http://www.my-auction.co.kr/auction/search.php');
+  const selectRegionMenu = await driver.findElement(By.id('address1_01'));
+  await selectRegionMenu.sendKeys('서울특별시');
+  const selectForSaleTypeCheckBox = await driver
+    .findElement(By.className('t_color01'))
+    .findElement(By.tagName('input'));
+  await selectForSaleTypeCheckBox.click();
+  const totaSearchButton = await driver
+    .findElement(By.id('search_btn'))
+    .findElement(By.tagName('button'));
+  await totaSearchButton.click();
+}
```

Comment:
```suggestion
  await driver.get('http://www.my-auction.co.kr/auction/search.php');
  const selectRegionMenu = await driver.findElement(By.id('address1_01'));
  
  await selectRegionMenu.sendKeys('서울특별시');
  const selectForSaleTypeCheckBox = await driver
    .findElement(By.className('t_color01'))
    .findElement(By.tagName('input'));
    
  await selectForSaleTypeCheckBox.click();
  const totaSearchButton = await driver
    .findElement(By.id('search_btn'))
    .findElement(By.tagName('button'));
    
  await totaSearchButton.click();
}
```

위에부터 계속 봤는데 코드들이 공백이 없어서 읽을때 눈이 좀 아픕니다... 같은 단위끼리 묶는게 가능할때는 묶어서 스페이스를 넣어주세요..ㅠㅠ